aten/src/ATen/native/native_functions.yaml

```diff
@@ -10965,10 +10917,6 @@
 
 - func: upsample_nearest3d.vec(Tensor input, int[]? output_size, float[]? scale_factors) -> Tensor
   python_module: nn
-  dispatch:
-    CPU: upsample_nearest3d_cpu
-    CUDA: upsample_nearest3d_cuda
-    QuantizedCPU: upsample_nearest3d_quantized_cpu
```

Comment:
There must be also a `upsample_nearest3d_quantized_cpu` to remove, right?

Reply:
Right you are, will remove that.